Makefile

```diff
@@ -0,0 +1,15 @@
+all: test
```

Comment:
What I've found hugely valuable with Makefiles is with the "immutability". `make clean` makes sure to reset the repo to its original state and testing becomes much more reliable when `node_modules/` gets wiped and re-installed from clean slate. This was the reason I started adding them to my repos. They give a really simple and clean set of instructions to run on a repo to make sure everything is working: `make clean && make` and all the expected things should've been run & built (dist builds, tests, installing deps, etc).

I would recommend the following approach: try it out, add it to the repos you work on and see if it is valuable for you. It might not make sense to add them *everywhere* but for something like this repo, or js-ipfs-api, I think they work great and give a really clean way to develop. YMMV. And I would imagine for newcomers it's easier to check which commands to run from a Makefile as opposed to package.json.

As mentioned above, Windows is the black sheep in this case too, so let's make sure any repo doesn't *depend* on Makefiles (ie. it's not the primary location to define the development tooling), they're additional info.